Linters-installation

MudasirSherwani opened the pull request.

topeogunleye reviewed (approved):
### Hi @MudasirSherwani, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
![congratulations](https://user-images.githubusercontent.com/52934128/163691476-087b4b9c-825b-40a3-85a1-d1bde44ffa53.gif)
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
> ## [Optional] suggestions:
## Because the previous reviewer did not mention this, I'm making this optional
> * [ ]  Please check my inline comments.
 >- [ ] From now on, I kindly suggest that you please make use of imperative mood in the subject line for your commit messages.
:x: Wrong
![image](https://user-images.githubusercontent.com/52934128/176625561-5a125f4b-4a93-4d1e-9daa-6b05dff142a2.png)
> ### Good Commit Messages:
>- [x]  Add fix for dark mode toggle state
>- [x] Update README file.
>- [x] Add styles to h1.
>
>- [ ] kindly remove your `node_modules` folder from your `linters-installation` GitHub branch as you may have mistakenly pushed  `node_modules` to your branch repo.
>- [ ] Please kindly add a more descriptive PR title.
> See a demo below:

https://user-images.githubusercontent.com/52934128/176308229-e31b2c8d-110b-4823-a2a8-31c1b7d7c73d.mp4


>- [ ] Please refer to this article: please read it and apply it in your work from now on: [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md);
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

MudasirSherwani commented:
### Additions

- install webhint
- install stylelint
- added Node Jason to gitingnore

### Changes

- change in head to remove h1
- added h1 in body tag
- deleted Node Module directory from All branches of gitHub
